core/templates/pages/skill-editor-page/skill-preview-tab/skill-preview-tab.component.html

```
@@ -7,23 +7,20 @@
           <div><span class="heading-secondary">Questions</span></div>
           <div *ngIf="questionsFetched">
             <div class="question-filters">
-              <input type="text"
-                     class="e2e-test-question-text-input"
-                     [(ngModel)]="questionTextFilter"
+              <input type="text" class="e2e-test-question-text-input" [(ngModel)]="questionTextFilter"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)